typescript/organisations.d.ts

```diff
@@ -226,3 +227,19 @@ export type ProductNotification = NewProductNotification & {
   createdAt: string
   updatedAt: string
 }
+
+export type NewAIDisclaimer = {
+  organisationId: string
+  type: 'FORMS_BUILDER'
+  acceptedBy: UserProfile
+  updatedBy?: UserProfile
```

Comment:
```suggestion
  updatedBy: UserProfile
```

This could just be the same as the person who accepted it on the create, to avoid dealing with optional property.